apps/client/src/components/VisualStyles.js

```diff
@@ -1,11 +1,12 @@
 import styled from "styled-components";
+import { Card } from "@datapunt/asc-ui";
 
 export const FigCaption = styled.figcaption`
   font-style: italic;
 `;
 
-export const Figure = styled.figure`
-  min-height: 1px; /* IE11 Bug */
+export const StyledCard = styled(Card)`
+  min-height: 100px; /* IE11 Bug */
```

Comment:
### This broke the IE11 fix

IE11 screenshot:

![image](https://user-images.githubusercontent.com/7781865/82800206-953a1880-9e7b-11ea-807f-2651872acd65.png)